task_02/tevfik_aksoy/main.cpp

```diff
@@ -0,0 +1,183 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <set>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+using std::get;
+using std::to_string;
+using std::vector;
+
+//Tuple of <int,int> for realization of vector of vectors of <int,int>
+typedef std::tuple<int, int> DestID_EdgeID;
+
+//Structure of nodes(vertices)
+struct Node
+{
+private:
+	int id = 0;
+
+public:
+	explicit Node(int id) : id(id) {}
+	explicit Node() : id(0) {}

```

Comment:
Вместо нескольких конструкторов достаточно сделать один с аргументом с дефолтным значением
```suggestion
      explicit Node(int id = 0) : id(id) {}
```

Reply:
`explicit` нужно указывать только для конструкторов с одним аргументом.